src/experimental/slam_toolbox_map_and_localization.cpp

```diff
@@ -26,21 +26,41 @@ MapAndLocalizationSlamToolbox::MapAndLocalizationSlamToolbox(rclcpp::NodeOptions
 : LocalizationSlamToolbox(options)
 /*****************************************************************************/
 {
+}
+
+/*****************************************************************************/
+void MapAndLocalizationSlamToolbox::configure()
+/*****************************************************************************/
+{
+  SlamToolbox::configure();
```

Comment:
I don't understand why there's both a configure and on_configure override

Reply:
I just reused the configure function, no particular reason.

<https://github.com/SteveMacenski/slam_toolbox/pull/561#discussion_r1055793020>  
>  Configure should just be on_configure now  

I see! I'll merge `configure` into `on_configure` .

Reply:
- [d76adf08a1c016da6424c98bca376ec7874a6145]
  - I have merged `setParams` and `configure` into `on_configure`.
    - related comment : <https://github.com/SteveMacenski/slam_toolbox/pull/561#discussion_r1055793020> <https://github.com/SteveMacenski/slam_toolbox/pull/561#discussion_r1055793266>  

minor change: Reverted smapper's configure to be called inside setParams. ( <https://github.com/SteveMacenski/slam_toolbox/commit/d76adf08a1c016da6424c98bca376ec7874a6145#diff-22e074ba7d16f1b0f47ce6b2b39fce9efbd5b685e3e312987240a444ec0bb7b3R334> )